compiler/onecc-docker/onecc-docker

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+''''exec "python3" "$0" "$@" #'''
+
+# Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+import subprocess
+import json
+import os
+import requests
+
+
+def _is_image(version) : 
+    cmd = ['docker', 'images', f'onecc:{version}']
+    p = subprocess.Popen(cmd,  stdout=subprocess.PIPE, stderr=subprocess.STDOUT) 
+    out, err = p.communicate()
+    dic = {}
+
+    decoded_out = [i.split() for i in out.decode().split("\n") if len(i)]
+
+    if len(decoded_out) <2 :
+        return False
+
+    dic[decoded_out[0][0]] = decoded_out[1][0]
+    dic[decoded_out[0][1]] = decoded_out[1][1]
+
+    tag_name = cmd[2].split(':')
+
+    if dic['REPOSITORY'] == tag_name[0] and dic['TAG'] == tag_name[1]  :
+        return True 
+    else :
+        return False
+
+
+
+def main():
```

Comment:
Unfortunately, I can't use the github rest api without authorization due to in-house network issue. So I suggest to support request header so that the users who need it can use it.

```
headers["Authorization"] = "token {}".format(/*token*/)
response = requests.get("https://api.github.com/repos/Samsung/ONE/tags", headers=headers)
```

https://docs.github.com/en/rest/overview/resources-in-the-rest-api#rate-limiting
https://docs.github.com/en/rest/overview/resources-in-the-rest-api#oauth2-token-sent-in-a-header